testsuite/integration/pom.xml

```diff
@@ -499,6 +499,12 @@
             <artifactId>resteasy-client</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-tracing-api</artifactId>
```

Comment:
@jamezp I'm not sure whether this is the correct place to put the necessary dependency. Could you please help to check this? Thanks!

Reply:
The version should definitely not be defined. But it depends on whether or not the dependency is used on the client side. If it is, then it's fine to define it here.

Reply:
If I remove the dep version then it said that it can't find the version, I'll check where the version is referenced from.

Reply:
version definition removed.

Reply:
This dependency should be moved to testsuite/integration/basic/pom.xml as its only use is to compile a test app in that module.